include/wx/xrc/xmlres.h

```diff
@@ -240,7 +240,7 @@ class WXDLLIMPEXP_XRC wxXmlResource : public wxObject
     // wxWindow::NewControlId(). Otherwise value_if_not_found is used.
     // Macro XRCID(name) is provided for convenient use in event tables.
     static int GetXRCID(const wxString& str_id, int value_if_not_found = wxID_NONE)
-        { return DoGetXRCID(str_id.mb_str(), value_if_not_found); }
+        { return DoGetXRCID(str_id.mb_str(wxConvLibc), value_if_not_found); }
```

Comment:
I think we should use `.utf8_str()` here: `str_id` comes from the program itself and there is no reason for it to be in the encoding of the current _execution_ environment, its encoding is determined at _compile_ time.